### DISCUSSION TITLE
Kotlin Multiplatform Mobile integration

### DISCUSSION THREAD

**Discussion by serhiynovos** [NativeScript/NativeScript#10027] [General]
Does anybody tried to integrate Kotlin Multiplatform Mobile with Nativescript application ?

**Comment by triniwiz** [comment]
I don't think anyone tried integrating it before.

**Reply by serhiynovos** [reply]
KMM is a very perspective technology, and it would be nice to have support for it because you can easily connect existing module as a submodule and use in application.

**Reply by triniwiz** [reply]
It should be possible to use it today but again I don't think anyone tried it before, android should handle it w/o issues.
